src/main/java/com/flowingcode/vaadin/addons/twincolgrid/TwinColGrid.java

```diff
@@ -78,6 +88,14 @@
 		Collection<T> getItems() {

```

Comment:
It should be possible to cast this as `List` (since we assume there is a `ListDataProvider`)